rtgui/batchqueue.cc

```diff
@@ -99,20 +100,18 @@ BatchQueue::~BatchQueue ()
     }
 
     fd.clear ();
-
-    delete pmenu;
 }
 
 void BatchQueue::resizeLoadedQueue()
 {
-    // TODO: Check for Linux
 #if PROTECT_VECTORS
     MYWRITERLOCK(l, entryRW);
 #endif
 
-    for (size_t i = 0; i < fd.size(); i++) {
-        fd.at(i)->resize(getThumbnailHeight());
-    }
+    const auto height = getThumbnailHeight ();
+
+    for (const auto entry : fd)
```

Comment:
Is it equivalent to the gone syntax ? Interesting !

On a side note, we have an official astyle format (provided in the root directory tree). The style says to add {} even on single line 'if' block. I know it's annoying, but I some point, someone will launch astyle and update the whole  code.